packages/synapse-react-client/src/lib/containers/entity/annotations/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -294,97 +284,69 @@ export const SchemaDrivenAnnotationEditor = (
               </Alert>
             )}
 
-            {submissionError && (
-              <Alert
-                dismissible={false}
-                show={showSubmissionError}
-                variant={'danger'}
-                transition={false}
-              >
+            {submissionError && showSubmissionError && (
+              <Alert severity="error">
                 Annotations could not be updated: {submissionError.reason}
               </Alert>
             )}
             {!formRefFromParent && (
               <>
                 <hr />
-                <div className="SaveButtonContainer">
-                  <Button
-                    variant="primary-500"
-                    onClick={() => {
+                <Box
+                  display="flex"
+                  justifyContent="space-between"
+                  sx={{ gridRowStart: 5 }}
+                >
+                  <ConfirmationButtons
+                    hasCancelButton={onCancel !== undefined}
+                    onCancel={() => {
+                      onCancel && onCancel()
+                    }}
+                    onConfirm={() => {
```

Comment:
I used the standard styling for the Cancel button (outlined), but I'm happy to update to use a contained button as in the original design, if that's preferred --

main | feature
:---:|:---:
<img width="1120" alt="main_SchemaDrivenAnnotationEditor_Buttons" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/1badcfa3-115d-47cc-97a8-b293fc9cc10e"> | <img width="1122" alt="feature_SchemaDrivenAnnotationEditor_Buttons" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/ad5b4c30-fc8a-4964-a770-5ae4d72df68f">